parts/linux/cloud-init/nodecustomdata.yml

```diff
@@ -344,11 +344,20 @@ write_files:
     current-context: localclustercontext
     #EOF
 
+{{if IsDynamicKubeletSupported}}
+- path: /etc/default/kubeletconfig.json
+  permissions: "0644"
+  owner: root
+  content: |
+    {{GetKubeletConfigFile .KubernetesConfig}}
+    #EOF
+{{end}}
+
```

Comment:
I guess you are moving this to cse_config?

Reply:
correct